src/snowflake/snowpark/_internal/utils.py

```diff
@@ -1280,3 +1281,19 @@ def import_or_missing_modin_pandas() -> Tuple[ModuleLikeObject, bool]:
         return modin, True
     except ImportError:
         return MissingModin(), False
+
+
+def is_feature_enabled_for_version(
+    session: "snowflake.snowpark.Session", parameter_name: str
+) -> bool:
+    """
```

Comment:
that doesn't look like a proper utility funciton to put in _internal/utils.py, maybe just addd this as an internal function for session.py

Reply:
good point, fixed